### ISSUE TITLE
UI back button inconsistent cursor [Mac OS]

### ISSUE THREAD

**Issue by peterblazejewicz** [balena-io/etcher#2849]
See the behaviour on the screen record:

![back-button](https://user-images.githubusercontent.com/14539/60769710-89166280-a0d3-11e9-9aee-253a10df1555.gif)

- **Etcher version:** 
1.5.51
- **Operating system and architecture:** 
Mac OS Mojave, 10.14.5 (18F132)
- **Image flashed:** 
n/a
- **Do you see any meaningful error information in the DevTools?** 
Yes, the detail is implemented using native button, but trying to force it acting like a hyperlink:

![image](https://user-images.githubusercontent.com/14539/60769700-64ba8600-a0d3-11e9-8a7d-9224ba6b9b5d.png)

**Comment by amrsekilly** [comment]
Can I open a PR with a fix for this?

**Comment by thundron** [comment]
@amrsekilly Indeed! PRs are more than welcome for anyone that wants to contribute

**Comment by amrsekilly** [comment]
Thanks, I'll submit a PR with a fix shortly!
